barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_format.test.cpp

```diff
@@ -674,7 +674,7 @@ TEST_F(AcirFormatTests, TestCollectsGateCounts)
     auto builder =
         create_circuit(constraint_system, /*size_hint*/ 0, witness, false, std::make_shared<bb::ECCOpQueue>(), true);
 
-    EXPECT_EQ(constraint_system.gates_per_opcode, std::vector<size_t>({ 2, 1 }));
+    EXPECT_EQ(constraint_system.gates_per_opcode, std::vector<size_t>({ 3, 1 }));
```

Comment:
increase because we add one more gate in constructor when we call `fix_witness` inside `put_constant_variable`